indexer/editable_map_object.cpp

```diff
@@ -312,6 +312,15 @@ bool EditableMapObject::ValidateEmail(string const & email)
   if (email.empty())
     return true;
 
+  if (strings::IsASCIIString(email))
+    return regex_match(email, regex("^[^@\\s]+@[\\w-]+(\\.[\\w-]+)+$"));
```

Comment:
So, my question is the same - why did you simplify the expression I asked you to use?